src/create-identity.js

```diff
@@ -1,16 +1,8 @@
-import publicKeyByPrivateKey from './public-key-by-private-key';
-
-import {
-    fromPrivate
-} from 'eth-lib/lib/account';
-import {
-    keccak256
-} from 'eth-lib/lib/hash';
-import Bytes from 'eth-lib/lib/bytes';
-
+import { ethers } from 'ethers';
```

Comment:
Should we also do `import { utils as ethersUtils } from 'ethers';`
so the bundler can proper tree shake?

Reply:
Done. Not sure this will help the bundle for tree shaking though 🤔